conbench/entities/history.py

```diff
@@ -32,7 +34,14 @@ class HistorySerializer:
     many = _Serializer(many=True)
 
 
-def get_history(case_id, context_id, hardware_hash):
+def get_history(case_id, context_id, hardware_hash, repo) -> List[tuple]:
+    """Given a case/context/hardware/repo, return all non-errored BenchmarkResults
+    (past, present, and future) on the default branch that match those criteria, along
+    with information about the stats of the distribution as of each BenchmarkResult.
+    Order is not guaranteed.
+
+    Primarily used to power the blue line in the timeseries plots.
+    """
```

Comment:
Love these doc strings! We don't need it now, but in a follow on we might extend this to _also_ include errored BenchmarkResults and display them with a 💣 or something in the plot itself

Reply:
Oh yeah that could be fun!

Reply:
Made #568